layouts/partials/navigation_level.html

```diff
@@ -0,0 +1,61 @@
+{{ $pages := .pages }}
```

Comment:
Is this supposed to be a rename of sidebar_level?

Reply:
It's modified version of sidebar_level made accordingly with new designs, I didn't delete sidebar_level, because I left in primary_top a way to use old sidebars.